ibm/mas_devops/roles/install_operator/templates/ibm-entitlement-with-artifactory.json.j2

```diff
@@ -2,4 +2,4 @@
 {% if artifactory_username is defined and artifactory_username != "" %}
 "wiotp-docker-local.artifactory.swg-devops.com":{"username":"{{ artifactory_username }}","password":"{{ artifactory_apikey }}","auth":"{{ (artifactory_username ~ ':' ~ artifactory_apikey) | b64encode }}"},
 {% endif %}
-"cp.icr.io/cp":{"username":"{{ icr_username }}","password":"{{ icr_password }}","auth":"{{ (icr_username ~ ':' ~ icr_password) | b64encode }}"}}}
+"{{ icr_host }}":{"username":"{{ icr_username }}","password":"{{ icr_password }}","auth":"{{ (icr_username ~ ':' ~ icr_password) | b64encode }}"}}}
```

Comment:
This was deliberately changed to be explicity ICR ... if we are using artifactory as the alternative source for ICR content then both credentials will be in the secret now .. this allows for mixed content (some from ICR and some from Artifactory) to work in the same namespace (e.g. using pre-release MAS code with GA truststore manager)